cli/Cargo.toml

```diff
@@ -79,6 +79,8 @@ tokio = { version = "0.2.22", features = ["full"] }
 tokio-rustls = "0.14.1"
 # Keep in-sync with warp.
 tokio-tungstenite = "0.11.0"
+# TODO(lucacasonato): go back to crates.io when ebkalderon/tower-lsp#243, ebkalderon/tower-lsp#244, and ebkalderon/tower-lsp#245 land, or when `lspower` is released.
+tower-lsp = { git = "https://github.com/lucacasonato/tower-lsp", rev = "0481ab5e092857dcca2f6854320a52a60738c3af" }
```

Comment:
Probably should consider waiting on this until these become true, IMO.

Reply:
Yes this is a merge blocker.